### PR TITLE
More clarification about performance numbers

### PR DESCRIPTION
**Pull request by rveerama1**
Please provide a description for what this PR is for.

Clarification about performance numbers.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by irisdingbj** [comment]
LGTM

**Review by craigbox — APPROVED** [review]

File: content/en/blog/2022/cryptomb-privatekeyprovider/index.md [hunk 1]
```diff
@@ -171,7 +171,11 @@ spec:
...
+The potential performance benefit depends on many factors. For example, the size of the cpuset Envoy is running on, incoming traffic pattern, encryption type (RSA or ECDSA), and key size. 
+
+Below, we show performance based on the total latency between k6, gateway and Fortio server. These show relative performance improvement using the CryptoMB provider, and are in no way representative of Istio's [general performance or benchmark results](/docs/ops/deployment/performance-and-scalability/).  Our measurements use different client tools (k6 and fortio), different setup (client, gateway and server running on separate nodes) and we create a new TLS handshake with every HTTP request. 
+
+We have [published a white paper](https://www.intel.com/content/www/us/en/architecture-and-technology/crypto-acceleration-in-xeon-scalable-processors-wp.html) with general crypographic performance numbers.
```
Comment (craigbox) [inline content/en/blog/2022/cryptomb-privatekeyprovider/index.md:178]:
```suggestion
We have [published a white paper](https://www.intel.com/content/www/us/en/architecture-and-technology/crypto-acceleration-in-xeon-scalable-processors-wp.html) with general cryptographic performance numbers.
```

File: content/en/blog/2022/cryptomb-privatekeyprovider/index.md [hunk 2]
```diff
@@ -171,11 +171,13 @@ spec:
...
 
-Istio ingress gateway TLS handshake performance comparison on 1.14-dev on May 10th 2022.
+Below, we show performance based on the total latency between k6, gateway and Fortio server. These show relative performance improvement using the CryptoMB provider, and are in no way representative of Istio's [general performance or benchmark results](/docs/ops/deployment/performance-and-scalability/).  Our measurements use different client tools (k6 and fortio), different setup (client, gateway and server running on separate nodes) and we create a new TLS handshake with every HTTP request.
 
-{{< image link="./istio-ingress-gateway-tls-handshake-perf-num.png" caption="Istio ingress gateway TLS handshake performance comparison" >}}
+We have [published a white paper](https://www.intel.com/content/www/us/en/architecture-and-technology/crypto-acceleration-in-xeon-scalable-processors-wp.html) with general cryptographic performance numbers.
+
+{{< image link="./istio-ingress-gateway-tls-handshake-perf-num.png" caption="Istio ingress gateway TLS handshake performance comparison; tested using 1.14-dev on May 10th 2022." >}}
```
Comment (craigbox) [inline content/en/blog/2022/cryptomb-privatekeyprovider/index.md:180]:
```suggestion
{{< image link="./istio-ingress-gateway-tls-handshake-perf-num.png" caption="Istio ingress gateway TLS handshake performance comparison. Tested using 1.14-dev on May 10th 2022." >}}
```

File: content/en/blog/2022/cryptomb-privatekeyprovider/index.md [hunk 3]
```diff
@@ -171,11 +171,13 @@ spec:
...
 
-Istio ingress gateway TLS handshake performance comparison on 1.14-dev on May 10th 2022.
+Below, we show performance based on the total latency between k6, gateway and Fortio server. These show relative performance improvement using the CryptoMB provider, and are in no way representative of Istio's [general performance or benchmark results](/docs/ops/deployment/performance-and-scalability/).  Our measurements use different client tools (k6 and fortio), different setup (client, gateway and server running on separate nodes) and we create a new TLS handshake with every HTTP request.
 
-{{< image link="./istio-ingress-gateway-tls-handshake-perf-num.png" caption="Istio ingress gateway TLS handshake performance comparison" >}}
+We have [published a white paper](https://www.intel.com/content/www/us/en/architecture-and-technology/crypto-acceleration-in-xeon-scalable-processors-wp.html) with general cryptographic performance numbers.
+
+{{< image link="./istio-ingress-gateway-tls-handshake-perf-num.png" caption="Istio ingress gateway TLS handshake performance comparison. Tested using 1.14-dev on May 10th 2022." >}}
```
Comment (craigbox) [inline content/en/blog/2022/cryptomb-privatekeyprovider/index.md:180]:
```suggestion
{{< image link="./istio-ingress-gateway-tls-handshake-perf-num.png" caption="Istio ingress gateway TLS handshake performance comparison. Tested using 1.14-dev on May 10th 2022" >}}
```

**Comment by rveerama1** [comment]
Thank you @craigbox

**Pull request merged**